gateway-api-spec.yaml

```diff
@@ -917,6 +917,9 @@ components:
         limit:
           description: The page size requested. The maximum value is 30 at present.
           type: integer
+        ascending:
+          description: Use ascending ordering of transactions.
```

Comment:
As a minor tweak, I think something like "If true, returns transactions in ascending order." reads a little more clearly :)